src/ripple/core/Config.h

```diff
@@ -203,7 +203,7 @@ class Config : public BasicConfig
     int PATH_SEARCH_MAX = 3;
 
     // Validation
-    std::optional<std::size_t>
+    std::optional<std::int64_t>
```

Comment:
I'd recommend leaving `VALIDATION_QUORUM` as an `std::size_t`, since negative values are not allowed and/or meaningful. It's a way of "self-documenting" the code.